woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -134,6 +134,17 @@ def test_unknown_inference(strings):
             assert isinstance(inferred_type, Unknown)
 
 
+def test_unknown_inference_all_null(nulls):
+    dtypes = ['object', 'string', 'category', 'datetime64[ns]']
+    if ks and isinstance(nulls[0], ks.Series):
+        dtypes = get_koalas_dtypes(dtypes)
+
+    for series in nulls:
+        for dtype in dtypes:
+            inferred_type = ww.type_system.infer_logical_type(series.astype(dtype))
+            assert isinstance(inferred_type, Unknown)
```

Comment:
we should also confirm that `inferred_type.transform(series)` works as expected for all of these series (though maybe that should go in `test_logical_types.py`)